otelib/backends/services/base.py

```diff
@@ -131,4 +131,4 @@ def _create_session(self) -> str:
                 f"{' content=' + str(response.content) if self.debug else ''}",
                 status=response.status_code,
             )
-        return json.loads(response.text)["session_id"]
+        return response.json()["session_id"]
```

Comment:
This syntax is a bit nicer and clarifies the move a bit to me. Still, not clear if its worth the cost

Reply:
Fully agree.